packages/features/ee/workflows/pages/workflow.tsx

```diff
@@ -277,7 +279,7 @@ function WorkflowPage() {
         }
         hideHeadingOnMobile
         heading={
-          session.data?.hasValidLicense &&
+          (IS_DEV || session.data?.hasValidLicense) &&
```

Comment:
We don't need the IS_DEV check. We an always show the save button, all we need is the banner

Reply:
This was added because:
1. On the local environment, session.data?.hasValidLicense evaluates to `false`, hence making the `heading` prop null.
2. The shell component only renders the CTA if the heading is not null.
3. Hence, in order to pass in the heading, this check was added.

reference: https://github.com/mhetreayush/cal.com/blob/22ccb69c6e5ba5d6ea24b24bd2535b2677abe8d9/packages/features/shell/Shell.tsx#L1030-L1066

Reply:
@CarinaWolli Let me know what you think about this.

Reply:
we can remove the complete `IS_DEV || session.data?.hasValidLicense` check. I'd love to make it look the same as for routing forms:
![Screenshot 2024-05-08 at 10 57 16 AM](https://github.com/calcom/cal.com/assets/30310907/66ffbfb0-d203-4a1f-91df-74f25c73e205)

Reply:
Sure thing